.github/workflows/build-native-release.yml

```diff
@@ -45,13 +45,25 @@ jobs:
       - name: Build wheel
         shell: bash
         run: ./ci/deploy.sh build_pyfory
+        env:
+          GITHUB_REF_NAME: ${{ github.ref_name }}
       - name: Install and verify wheel
         shell: bash
         run: |
           python -m pip install --upgrade pip
           pip install dist/*.whl
-          python -c "import pyfory; print(pyfory.__version__)"
-      - name: Upload artifacts
+          INSTALLED_VERSION=$(python -c "import pyfory; print(pyfory.__version__)")
+          echo "Installed version: $INSTALLED_VERSION"
+          # Verify version matches the tag (strip leading 'v' if present)
+          EXPECTED_VERSION="${{ github.ref_name }}"
+          EXPECTED_VERSION=${EXPECTED_VERSION#v}
```

Comment:
this version seems not right too, maybe need same logic to replace `-alpha/beta/rc`?